buttplug/buttplug-device-config/buttplug-device-config.json

```diff
@@ -3869,6 +3869,10 @@
             "tx": "0000fe01-0000-1000-8000-00805f9b34fb",
             "rx": "0000fe02-0000-1000-8000-00805f9b34fb"
           },
+          "0000fffe-0000-1000-8000-00805f9b34fb": {
```

Comment:
The changes to the yaml need to be included too: the JSON is generated from the yaml via `npm run build`

Reply:
Fixed in e9a8f3cc01bad49bfc8523f5d84091b383043aaf.
Completely missed the yaml file.